app/components/glimmer/route-graphic.js

```diff
@@ -3,59 +3,67 @@ import { cached } from '@glimmer/tracking';
 import Graphic from '@arcgis/core/Graphic';
 import Polyline from '@arcgis/core/geometry/Polyline';
 import SimpleLineSymbol from '@arcgis/core/symbols/SimpleLineSymbol';
-import { registerDestructor } from '@ember/destroyable';
 
 export default class RouteGraphicComponent extends Component {
+  createGraphic = () => {
+    console.log('updating graphic');
 
-  constructor() {
-    super(...arguments);
-
-    registerDestructor(this, this.destroyGraphic);
-  }
-
-  @cached
-  get graphic() {
```

Comment:
this was the main issue -- `graphic` is getting obliterated all the time -- potentially because `routeData.color` is changing